File: C++/1012 - Area.cpp
#include <iostream>
#include <cmath>

void triangulo(double A, double C)
{
    double area = ((A*C)/2.0);
    printf("TRIANGULO: %.3f\n", area);
}

void circulo(double C)
{
    double pi = 3.14159;
    double area = pi*(pow(C,2));
    printf("CIRCULO: %.3f\n", area);
}

void trapezio(double A, double B, double C)
{
    double area = ((A+B)*C)/2.0;
    printf("TRAPEZIO: %.3f\n", area);
}

void quadrado(double B)
{
    double area = pow(B,2);
    printf("QUADRADO: %.3f\n", area);
}

void retangulo(double A, double B)
{
    double area = A*B;
    printf("RETANGULO: %.3f\n", area);
}

int main()
{
   double A,B,C;
   scanf("%lf %lf %lf",&A,&B,&C);
   triangulo(A,C);
   circulo(C);
   trapezio(A,B,C);
   quadrado(B);
   retangulo(A,B);
   return 0;
}

File: Python REDUX/1012 - Area.py
import math

def triangulo(base, altura):
    area = (base*altura)/2
    print("TRIANGULO: %.3f" % area)

def circulo(raio):
    pi = 3.14159
    area = pi*math.pow(raio,2)
    print("CIRCULO: %.3f" % area)
    
def trapezio(base_1, base_2, altura):
    area = ((base_1+base_2)*altura)/2
    print("TRAPEZIO: %.3f" % area)

def quadrado_ou_retangulo(lado_1, lado_2, tipo):
    area = lado_1*lado_2
    print("%s: %.3f" % (tipo, area))

if __name__ == "__main__":
    nums = input().split() # A, B, C
    nums = [float(num) for num in nums]
    
    triangulo(nums[0], nums[2])
    circulo(nums[2])
    trapezio(nums[0], nums[1], nums[2])
    quadrado_ou_retangulo(nums[1],nums[1], "QUADRADO")
    quadrado_ou_retangulo(nums[0],nums[1], "RETANGULO")